src/components/atoms/Info.tsx

```diff
@@ -11,7 +11,7 @@ export const Info = ({ description }: { description: string }) => {
       ref={ref}
       className="my-auto flex ml-1"
       onMouseEnter={() => {
-        updateTooltip({ description, ref, facing: 'bottom'});
+        updateTooltip({ description, ref, facing: 'bottom' });
```

Comment:
Does this extra white space need to be there?

Reply:
yup! It should be there in fact. (although it's just consistency and doesn't have any actual impact)